### PR TITLE
Gui

### PR DESCRIPTION
**Pull request by Moussiaa**

### CODE REVIEW

File: src/main/java/adry/projectile/ProjectileFrame.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        // add listeners
+        velocityField.getDocument().addDocumentListener(new SimpleDocumentListener() {
+            @Override
+            public void update(DocumentEvent e) {
+                calculate();
+            }
+        });
+
+        // slider
+        angleAdjuster.getChangeListeners();
```
Comment (andrewoid) [inline src/main/java/adry/projectile/ProjectileFrame.java]: What does this line do?

File: src/main/java/adry/projectile/ProjectileFrame.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                                Double.parseDouble(velocityField.getText()),
+                                angleAdjuster.getValue()
+                        );
+                        projectile.setSeconds(
+                                Double.parseDouble(secondsField.getText())
+                        );
+                        interceptXLabel.setText(Double.toString(projectile.getX()));
+                        peakYLabel.setText(Double.toString(projectile.getY()));
+                        yField.setText(Double.toString(projectile.getPeakY()));
+                        xField.setText(Double.toString(projectile.getInterceptX()));
```
Comment (andrewoid) [inline src/main/java/adry/projectile/ProjectileFrame.java]: call calculate()

File: src/main/java/adry/projectile/ProjectileFrame.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                                Double.parseDouble(velocityField.getText()),
+                                angleAdjuster.getValue()
+                        );
+                        projectile.setSeconds(
+                                Double.parseDouble(secondsField.getText())
+                        );
+                        interceptXlabel.setText(Double.toString(projectile.getX()));
+                        peakYlabel.setText(Double.toString(projectile.getY()));
+                        fieldY.setText(Double.toString(projectile.getPeakY()));
+                        fieldX.setText(Double.toString(projectile.getInterceptX()));
```
Comment (andrewoid) [inline src/main/java/adry/projectile/ProjectileFrame.java:116]: you should be calling calculate() like in the other Listeners

**Review by andrewoid — APPROVED** [review]

File: src/main/java/adry/projectile/ProjectileFrame.java (hunk 3 above)
Comment (Moussiaa) [reply src/main/java/adry/projectile/ProjectileFrame.java:116]: should I switch to branch gui to commit this change or stay on graph?

Comment (andrewoid) [reply src/main/java/adry/projectile/ProjectileFrame.java:116]: Did it on graph.

But merge the PR.

**Pull request merged**